### PR TITLE
convertConfig Tests

### PR DESCRIPTION
**Pull request by jalextowle**
Added tests for the `convertConfig` function that use the new framework.

### CODE REVIEW

File: browser/webpack.config.js [hunk 1]
```diff
@@ -2,6 +2,9 @@ const path = require('path');
 
 module.exports = {
     entry: './conversion-tests/conversion_test.ts',
+    node: {
+      fs: 'empty'
+    },
```
Comment (fabioberger) [inline browser/webpack.config.js]: Why was this necessary?

File: rpc/clients/typescript/test/ws_client_test.ts [hunk 2]
```diff
@@ -445,10 +444,11 @@ blockchainTests.resets('WSClient', env => {
                                     }
                                 `;
                                 connection.sendUTF(response);
-                                numMessages++;
-                                return;
                             }
-                            numMessages++;
+                            if (++numMessages > 2) {
+                                // NOTE(jalextowle): This log is here to help debug intermittent failures in CI.
+                                console.log('extra message', message); // tslint:disable-line:no-console
```
Comment (fabioberger) [inline rpc/clients/typescript/test/ws_client_test.ts]: Should be resolved in: https://github.com/0xProject/0x-mesh/pull/705

File: browser/conversion-tests/conversion_test.ts [hunk 3]
```diff
@@ -139,6 +146,51 @@ WebAssembly.instantiateStreaming(fetch('conversion_test.wasm'), go.importObject)
...
     testValidationResults(validationResults);
 
+    // Execute the Typescript --> Go tests
+    const ethereumRPCURL = 'http://localhost:8545';
+
+    // Set up a Web3 Provider that uses the RPC endpoint
+    // tslint:disable:no-object-literal-type-assertion
+    const provider = new Web3ProviderEngine();
+    provider.addProvider(new RPCSubprovider(ethereumRPCURL));
+    provider.start();
```
Comment (albrow) [inline browser/conversion-tests/conversion_test.ts]: I don't think you need to call `provider.start()` here.

```suggestion
```

File: browser/go/conversion-test/conversion_test.go [hunk 4]
```diff
@@ -41,7 +41,7 @@ func TestBrowserConversions(t *testing.T) {
```
Comment (albrow) [inline browser/go/conversion-test/conversion_test.go]: `TestBrowserConversions` took 70 seconds for me the first time I ran it because I needed to download new packages from NPM. Can you change the context timeout to 120 seconds?

**Comment by albrow** [comment]
Just some small suggestions. Looks pretty good overall.

File: browser/go/conversion-test/conversion_test.go (hunk 4 above)
Comment (jalextowle) [reply browser/go/conversion-test/conversion_test.go]: Yep

File: browser/webpack.config.js (hunk 1 above)
Comment (jalextowle) [reply browser/webpack.config.js]: For some reason, we get an error that is close to "cannot resolve \"fs\" in @0x/dev-utils". After doing some reading, this is actually a bug in webpack and is fixed by adding this line. This is the issue I referred to find this solution: https://github.com/webpack-contrib/css-loader/issues/447. After reading the issue again, I'm going dig into this a bit more on a low-priority cycle, because it seems like there is a better way to fix this.
